README.rst

```diff
@@ -1013,6 +1013,17 @@ Choose from the list of available rules:
 
   The correct case must be used for standard PHP types in phpdoc.
 
+* **phpdoc_types_order**
+
+  Sorts PHPDoc types.
+
+  Configuration options:
+
+  - ``null_position`` (``'always_first'``, ``'always_last'``, ``'default'``): forces the
```

Comment:
`default` keeps the position of `null` as defined by the sort algorithm, unlike `always_first` and `always_last` which override it.